tests/helper_functions.py

```diff
@@ -635,3 +636,28 @@ def clear_hub_cache():
     except ImportError:
         # Local import check for mlflow-skinny not including huggingface_hub
         pass
+
+
+def flaky(max_tries=3):
```

Comment:
Good idea refactoring this utility test function to the helper_functions file. +1